lib/fs_utils/generate.js

```diff
@@ -188,6 +188,11 @@ const optimize = (data, map, path, optimizers, sourceFiles) => {
   return promiseReduce(optimizers, runOptimizer, initial);
 };
 
+const getInlineSourceMapUrl = map => {
+  const base64String = new Buffer(JSON.stringify(map)).toString('base64');
+  return `data:application/json;charset=utf-8;base64,${base64String}`;
```

Comment:
Are you sure? I don't see a default value for the charset in the [RFC](https://tools.ietf.org/html/rfc2397). I'll admit I only added it because the `inline-source-map` package [also specifies it](https://github.com/thlorenz/inline-source-map/blob/6e089385f363456c7d9ac6fb6fbce4a5516f3382/index.js#L104).